packages/tokens/src/rwa/identity_registry_storage/storage.rs

```diff
@@ -0,0 +1,600 @@
+/// # Errors
+///
+/// * [`IRSError::IdentityNotFound`] - If no identity is found for the
+///   `account`.
+///
+/// # Events
+///
+/// * topics - `["identity_modified", old_identity: Address, new_identity:
+///   Address]`
+/// * data - `[]`
+///
+/// # Security Warning
+///
+/// **IMPORTANT**: This function bypasses authorization checks and should only
+/// be used:
+/// - During contract initialization/construction
+/// - In admin functions that implement their own authorization logic
+///
+/// Using this function in public-facing methods may create significant security
+/// risks as it could allow unauthorized modifications.
+pub fn modify_identity(e: &Env, account: &Address, new_identity: &Address) {
+    let key = IRSStorageKey::Identity(account.clone());
+
+    let old_identity: Address = e
+        .storage()
+        .persistent()
+        .get(&key)
+        .unwrap_or_else(|| panic_with_error!(e, IRSError::IdentityNotFound));
+
+    e.storage().persistent().set(&key, new_identity);
+
+    emit_identity_modified(e, &old_identity, new_identity);
+}
+
+/// Removes an identity and all associated country data.
+///
+/// # Arguments
+///
+/// * `e` - The Soroban environment.
+/// * `account` - The account address whose identity is being removed.
+///
+/// # Errors
+///
+/// * [`IRSError::IdentityNotFound`] - If no identity is found for the
+///   `account`.
+///
+/// # Events
+///
+/// * topics - `["identity_unstored", account: Address, identity: Address]`
+/// * data - `[]`
+///
+/// Emits for each country data removed:
+/// * topics - `["country_removed", account: Address, country_relation:
+///   CountryRelation]`
+/// * data - `[]`
+///
+/// # Security Warning
+///
+/// **IMPORTANT**: This function bypasses authorization checks and should only
+/// be used:
+/// - During contract initialization/construction
+/// - In admin functions that implement their own authorization logic
+///
+/// Using this function in public-facing methods may create significant security
+/// risks as it could allow unauthorized modifications.
+pub fn remove_identity(e: &Env, account: &Address) {
+    let identity_key = IRSStorageKey::Identity(account.clone());
+
+    let identity: Address = e
+        .storage()
+        .persistent()
+        .get(&identity_key)
+        .unwrap_or_else(|| panic_with_error!(e, IRSError::IdentityNotFound));
+    e.storage().persistent().remove(&identity_key);
+
+    emit_identity_unstored(e, account, &identity);
+
+    // Remove all associated identity profile
+    let profile_key = IRSStorageKey::IdentityProfile(account.clone());
+    let profile: IdentityProfile =
+        e.storage().persistent().get(&profile_key).expect("identity profile must be already set");
+    e.storage().persistent().remove(&profile_key);
+
+    for country_data in profile.countries {
+        emit_country_data_event(e, CountryDataEvent::Removed, account, &country_data);
+    }
+}
+
+/// Adds multiple country data entries to an existing identity.
```

Comment:
It is mentioned above in the library level inline docs, but I still think including the statement "does not check for duplicate countries" here is valuable